google-codelab-step/google_codelab_step_test.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+goog.module('googlecodelabs.CodelabStepTest');
+goog.setTestOnly();
+
+const CodelabSurvey = goog.require('googlecodelabs.CodelabStep');
+const testSuite = goog.require('goog.testing.testSuite');
+goog.require('goog.testing.asserts');
+goog.require('goog.testing.jsunit');
+
+testSuite({
+  testMoo() {
+      assertTrue(false);
```

Comment:
I'm planning to make our CI actually run tests instead of `//demo:hello_test`. When I do that, this will start failing for obvious reasons. Maybe at least change it to `true`, please? :)